Vagrant/bootstrap.sh

```diff
@@ -215,7 +276,29 @@ if [ "$app_env" == "production" ]; then
     $RUN_AS_INSTALLUSER RAILS_ENV=production bundle exec rake assets:precompile
     $RUN_AS_INSTALLUSER RAILS_ENV=production bundle exec rake datarepo:setup_defaults
 fi
+# Fix up configuration files
+# 1. FITS
+$RUN_AS_INSTALLUSER sed -i "s@config.fits_path = \".*\"@config.fits_path = \"$fitsdir/$fitsver/fits.sh\"@" config/initializers/sufia.rb
+# 2. Create Sufia Solr core
+cd $SOLR_DATA
+$RUN_AS_SOLR_USER mkdir -p $app_env/conf # We name the cores after their environment
+$RUN_AS_SOLR_USER echo "name=$app_env" > $app_env/core.properties
+install -o $SOLR_USER -m 444 $hydradir/solr_conf/conf/solrconfig.xml $app_env/conf/solrconfig.xml
+install -o $SOLR_USER -m 444 $hydradir/solr_conf/conf/schema.xml $app_env/conf/schema.xml
+# Make links to keep the Hydra Solr solrconfig.xml paths happy
+$RUN_AS_SOLR_USER ln -s $SOLR_INSTALL/solr/contrib
+$RUN_AS_SOLR_USER ln -s $SOLR_INSTALL/solr/dist
+$RUN_AS_SOLR_USER mkdir lib
+$RUN_AS_SOLR_USER ln -s $SOLR_INSTALL/solr/contrib lib/contrib
+cd $hydradir
+# 3. Make the solr.yml file point to an appropriate $app_env core
+sed -i '/production:/ {N; s@^production:.*development@production:\n  url: http://localhost:8983/solr/production@}' config/solr.yml
+# 4. Make the blacklight.yml file point to an appropriate $app_env core
+sed -i '/production:/ {N; N; s@^production:\(.*\)/development@production:\1/production@}' config/blacklight.yml
```

Comment:
This hurt my head looking at it the first time, I think I understand it now. It's changing the production urls to point to the production cores, right?